scripts/sri.R

```diff
@@ -0,0 +1,289 @@
+source("scripts/etl/fetch-data.R")
+source("scripts/functions/sri.R")
+
+library(future.apply)
+
+# Set the number of cores for parallel processing to 75% of available cores
+num_cores <- parallel::detectCores()*0.75
```

Comment:
Does this work without floor()? Wouldn't `num_cores = floor(parallel::detectCores()*0.75)` be better?

Reply:
You're absolutely right, thank you for pointing this out. It has been addressed in commit [f30efe5](https://github.com/Sage-Bionetworks/recover-sleep-measures/pull/4/commits/f30efe5fa36ac68e0cd5ea4f34434c4611993e98).